models/books-sequelize.mjs

```diff
@@ -20,12 +21,14 @@ async function connectDB() {
     author: Sequelize.DataTypes.STRING,
     publication_date: Sequelize.DataTypes.DATE,
     abstract: Sequelize.DataTypes.TEXT,
-    cover: Sequelize.DataTypes.STRING
+    cover: Sequelize.DataTypes.STRING,
+    userId: Sequelize.DataTypes.STRING
   }, {
     sequelize,
     modelName: 'Books',
     timestamps: false
   });
+  // SQBook.belongsTo(SQUser);
```

Comment:
remove it if you no longer need it

Reply:
Sure, just seems that it does not remove but in the next features I already remove it